packages/action-menu/stories/index.ts

```diff
@@ -47,7 +49,10 @@ export const ActionMenuMarkup = ({
                     : (staticValue as 'black' | 'white')
             )}
             size=${size}
-            @change="${changeHandler}"
+            @change=${(event: Event & { target: ActionMenu }) => {
+                changeHandler(event.target.value);
+                onChange(event.target.value);
```

Comment:
Keeping both of these on purpose? Backwards compatibility?

Reply:
Yeah. I think one of the passes of the docs site would be specifically targeted to simplifying and normalizing this.